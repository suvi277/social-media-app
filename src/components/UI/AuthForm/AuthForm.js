import React from 'react';
import './AuthForm.scss';

const AuthForm = ({ title, children }) => {
	return (
		<div className="auth-wrapper">
			<h2>{title}</h2>
			{children}
		</div>
	);
};

export { AuthForm };
