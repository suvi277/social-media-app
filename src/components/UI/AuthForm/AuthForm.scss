.auth-wrapper {
    margin: 0 auto;
    border: 1px solid #ccc;
    padding: 20px;

    @media (min-width: 768px) {
        width: 350px;
    }
}