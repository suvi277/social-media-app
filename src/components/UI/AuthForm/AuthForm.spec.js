import React from 'react';
import { shallow } from 'enzyme';
import { AuthForm } from './AuthForm';

describe('AuthForm Component', () => {
	const wrapper = shallow(
		<AuthForm title="Sign In">
			<div>Its children</div>
		</AuthForm>
	);

	it('renders as expected', () => {
		expect(wrapper).toMatchSnapshot();
	});
	it('should render a h2 tag', () => {
		expect(wrapper.find('h2')).toHaveLength(1);
	});
	it('should show the ', () => {
		expect(wrapper.find('h2').text()).toBe('Sign In');
	});
});
