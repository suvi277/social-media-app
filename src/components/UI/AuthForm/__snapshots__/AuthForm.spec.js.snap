// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuthForm Component renders as expected 1`] = `ShallowWrapper {}`;
